index.rst

```diff
@@ -41,4 +41,5 @@ Other content is not visible in A+.
   moodle_astra/index
   active_elements/index
   point_of_interest/index
+  jutut/index
```

Comment:
I think I could move the Jutut module a bit higher (earlier in the manual) so that it is easier to see in the table of contents.